RubeesEat.IntegrationTests/WebTesting/WebTestElementFinder.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.ObjectModel;
+using OpenQA.Selenium;
+using RubeesEat.Exceptions;
+using RubeesEat.IntegrationTests.WebTesting;
+
+namespace RubeesEat.IntegrationTests.PageObjects;
+
+public static class WebTestElementFinder
+{
+    public static IWebElement FindElement(ISearchContext context, string id)
+    {
+        ReadOnlyCollection<IWebElement> elements = context.FindElements(By.CssSelector($"[data-test-id='{id}']"));
+        if (elements.Count == 0)
+            throw new NoSuchElementException($"Element with id: {id} was not found.");
```

Comment:
I would emphasize the fact that we are looking for a "test id" and not an HTML id. I would also suggest putting the parameter in single quotes to make the id more apparent in the output.

Please fix this in the exceptions below as well. For the action stuff we should also emphasize that we are missing a "test action"